app/src/main/java/com/gogiyumandroid/RestaurantAdapter/RestaurantAdapter.kt

```diff
@@ -0,0 +1,78 @@
+package com.gogiyumandroid.RestaurantAdapter
+
+//class RestaurantAdapter(val _name: String, val _address: String, val _foods: Array<String> ) {
+class RestaurantAdapter(val _name: String, val _address: String, val _yrating: Double, val _grating: Double,
```

Comment:
Can you explain how this class works?

Reply:
After I get datas from url, I store datas as class form.

Reply:
Is this for tableview?

Reply:
yes